src/components/TodoList.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+type Props = {
+  preparedTodos: Todo[],
+  onDelete: (id: number) => void;
+  tempTodo: Todo | null,
+  updateTodo: (uTodo: Todo) => void,
+  loadingTodoIds: number[]
+};
+
+export const TodoList: React.FC<Props> = ({
+  preparedTodos,
+  onDelete = () => { },
+  tempTodo,
+  updateTodo = () => { },
```

Comment:
Default function for updateTodo prop is provided in the component definition. Similar to the onDelete prop, this is unnecessary and should be passed from the parent. Remove the default function definition.